media/css/firefox/new/compare.scss

```diff
@@ -0,0 +1,487 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import "../../../../node_modules/@mozilla-protocol/core/protocol/css/includes/lib";
+@import "../../../../node_modules/@mozilla-protocol/core/protocol/css/components/button";
+@import "../../../../node_modules/@mozilla-protocol/core/protocol/css/components/download-button";
+@import "../../../../node_modules/@mozilla-protocol/core/protocol/css/components/call-out";
+@import "../../../../node_modules/@mozilla-protocol/core/protocol/css/components/hero";
+@import "../../../../node_modules/@mozilla-protocol/core/protocol/css/components/feature-card";
```

Comment:
I don't think you're using feature card anymore.

Reply:
It's used on v1

Reply:
Ah, yeah.